### DISCUSSION TITLE
How to disasble (or make it static) buffeing animation?

### DISCUSSION THREAD

**Discussion by bheeshmpita** [tomasklaen/uosc#858] [Q&A] [answered]
Is it possible to make buffering animation a static image else disable it completely? How can i do so?
Thanks

**Comment by hooke007** [comment, marked as answer]
https://github.com/tomasklaen/uosc/blob/35699d300449b51d79bb9e2397a6de540bf108df/src/uosc.conf#L233

**Reply by bheeshmpita** [reply]
any static image also possible?

**Reply by hooke007** [reply]
hardcoded in the script.
https://github.com/tomasklaen/uosc/blob/35699d300449b51d79bb9e2397a6de540bf108df/src/uosc/elements/BufferingIndicator.lua#L30-L37
